invalid main.ogc_server url break the build

output:
```
Get DescribeFeatureType for url: http://plan.epfl.ch/wms_themes?VERSION=1.1.0&REQUEST=DescribeFeatureType&SERVICE=WFS
Traceback (most recent call last):
  File ".build/venv/bin/pot-create", line 11, in <module>
    sys.exit(main())
  File "/var/www/vhosts/planv4/private/epfl_authgeoportail/.build/venv/lib/python2.7/site-packages/lingua/extract.py", line 330, in main
    for message in extractor(real_filename, options):
  File "/var/www/vhosts/planv4/private/epfl_authgeoportail/.build/venv/lib/python2.7/site-packages/c2cgeoportal/lib/lingua_extractor.py", line 210, in __call__
    self._import(LayerWMS, messages, self._import_layer_wms)
  File "/var/www/vhosts/planv4/private/epfl_authgeoportail/.build/venv/lib/python2.7/site-packages/c2cgeoportal/lib/lingua_extractor.py", line 240, in _import
    callback(item, messages)
  File "/var/www/vhosts/planv4/private/epfl_authgeoportail/.build/venv/lib/python2.7/site-packages/c2cgeoportal/lib/lingua_extractor.py", line 249, in _import_layer_wms
    url, wms_layer, layer.item_type, layer.name, layer.id, messages
  File "/var/www/vhosts/planv4/private/epfl_authgeoportail/.build/venv/lib/python2.7/site-packages/c2cgeoportal/lib/lingua_extractor.py", line 293, in _import_layer_attributes
    for attribute in self._layer_attributes(url, layer):
  File "/var/www/vhosts/planv4/private/epfl_authgeoportail/.build/venv/lib/python2.7/site-packages/c2cgeoportal/lib/lingua_extractor.py", line 346, in _layer_attributes
    describe = parseString(content)
  File "/usr/lib64/python2.7/xml/dom/minidom.py", line 1931, in parseString
    return expatbuilder.parseString(string)
  File "/usr/lib64/python2.7/xml/dom/expatbuilder.py", line 940, in parseString
    return builder.parseString(string)
  File "/usr/lib64/python2.7/xml/dom/expatbuilder.py", line 223, in parseString
    parser.Parse(string, True)
xml.parsers.expat.ExpatError: not well-formed (invalid token): line 1131, column 22
make: *** [epfl_authgeoportail/locale/epfl_authgeoportail-client.pot] Error 1

```

--- a/c2cgeoportal/lib/lingua_extractor.py
+++ b/c2cgeoportal/lib/lingua_extractor.py
@@ -37,6 +37,7 @@ import traceback
 from json import loads
 from urlparse import urlsplit
 from xml.dom.minidom import parseString
+from xml.parsers.expat import ExpatError
 from sqlalchemy.exc import ProgrammingError, NoSuchTableError
 from sqlalchemy.orm.exc import NoResultFound
 from sqlalchemy.orm.util import class_mapper
@@ -347,6 +348,13 @@ class GeoMapfishThemeExtractor(Extractor):  # pragma: no cover
             try:
                 describe = parseString(content)
                 self.featuretype_cache[url] = describe
+            except ExpatError as e:
+                print(
+                    "WARNING! an error occured while trying to "
+                    "parse the DescribeFeatureType document."
+                )
+                print(str(e))
+                print("URL: {0!s}\nxml:\n{1!s}".format(url, content))
             except AttributeError:
                 print(
                     "WARNING! an error occured while trying to "

